org.compiere.db.oracle.provider/src/org/compiere/db/DB_Oracle.java

```diff
@@ -355,7 +355,7 @@ public String getStatus()
             sb.append(" , # Min Pool Size: ").append(m_ds.getMinPoolSize());
             sb.append(" , # Max Pool Size: ").append(m_ds.getMaxPoolSize());
             sb.append(" , # Max Statements Cache Per Session: ").append(m_ds.getMaxStatementsPerConnection());
-            sb.append(" , # Active Transactions: ").append(Trx.getActiveTransactions().length);
+            sb.append(" , # Open Transactions: ").append(Trx.getActiveTransactions().length);

```

Comment:
probably should change to getOpenTransactions() here ?

Reply:
ready - method name was wrong in Trx - fixed and deprecated the method with wrong name